core/drivers/stm32_gpio.c

```diff
@@ -900,16 +920,44 @@ static TEE_Result stm32_pinctrl_dt_get(struct dt_pargs *pargs,
 }
 #endif /*CFG_DRIVERS_PINCTRL*/
 
+static void __unused stm32_gpio_get_conf_sec(struct stm32_gpio_bank *bank)
+{
+	if (bank->sec_support) {
```

Comment:
looking back at these APIs, I'll change the patch so that I use the existing ones that configure each pin instead of applying a mask on the bank.